base/loading.jl

```diff
@@ -2787,6 +2791,32 @@ function show(io::IO, cf::CacheFlags)
     print(io, ", opt_level = ", cf.opt_level)
 end
 
+# Set by FileWatching.__init__()
+const mkpidlock_hook = Ref{Function}()
+const trymkpidlock_hook = Ref{Function}()
+const parse_pidfile_hook = Ref{Function}()
```

Comment:
```suggestion
global mkpidlock_hook
global trymkpidlock_hook
global parse_pidfile_hook
```
There is no reason to add overhead here and make is slower. You can directly assign to globals in other modules now.